### ISSUE TITLE
Error on make install on cygwin

### ISSUE THREAD

**Issue by kieranshanley** [JuliaLang/julia#48658]
Hi there,

I am trying to build and install julia on cygwin. I checked out the source code from the github repo and ran

make;
make install

Got all the way to cache stdlibs and then generated the following error:

# Cache stdlibs
    ESC[32;1mJULIAESC[0m ESC[37;1minstallESC[0m
ERROR: SystemError: opening file "\\home\\shanleyk1\\julia\\contrib\\cache_stdlibs.jl": No such file or directory
Stacktrace:
  [1] systemerror(p::String, errno::Int32; extrainfo::Nothing)
    @ Base .\error.jl:176
  [2] kwcall(::NamedTuple{(:extrainfo,), Tuple{Nothing}}, ::typeof(systemerror), p::String, errno::Int32)
    @ Base .\error.jl:176
  [3] kwcall(::NamedTuple{(:extrainfo,), Tuple{Nothing}}, ::typeof(systemerror), p::String)
    @ Base .\error.jl:176
  [4] #systemerror#83
    @ .\error.jl:175 [inlined]
  [5] systemerror
    @ .\error.jl:175 [inlined]
  [6] open(fname::String; lock::Bool, read::Nothing, write::Nothing, create::Nothing, truncate::Nothing, append::Nothing)
    @ Base .\iostream.jl:293
  [7] open
    @ .\iostream.jl:275 [inlined]
  [8] open(f::Base.var"#419#420"{String}, args::String; kwargs::Base.Pairs{Symbol, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
    @ Base .\io.jl:393
  [9] open
    @ .\io.jl:392 [inlined]
 [10] read
    @ .\io.jl:473 [inlined]
 [11] _include(mapexpr::Function, mod::Module, _path::String)
    @ Base .\loading.jl:1936
 [12] include(mod::Module, _path::String)
    @ Base .\Base.jl:457
 [13] exec_options(opts::Base.JLOptions)
    @ Base .\client.jl:307
 [14] _start()
    @ Base .\client.jl:522
make: *** [Makefile:281: install] Error 1

Cheers,

Kieran

**Comment by inkydragon** [comment]
- build commit hash?
- `julia\contrib\cache_stdlibs.jl` Does this file really not exist?

**Comment by kieranshanley** [comment]
Good point I should have included the hash. The file does exist. It seems
to be an issue with escaping the path separator.

Cheers

Kieran

**Comment by N5N3** [comment]
We miss some `call cygpath_w` here.

**Comment by stahta01** [comment]
This problem was fixed for me when I tried it yesterday with the git master branch.

I did have to copy the mingw import library to get cygwin to build.

Tim S.

**Comment by gitboy16** [comment]
I guess this can be closed now?
